deepchem/models/torch_models/ferminet.py

```diff
@@ -42,48 +90,41 @@ class Ferminet:
     def __init__(
         self,
         nucleon_coordinates: List[List],
-        spin: float,
-        charge: int,
+        spin: int,
+        ion_charge: int,
         seed: Optional[int] = None,
         batch_no: int = 10,
+        pretrain=True,
     ):
         """
-        Parameters:
-        -----------
-        nucleon_coordinates:  List[List]
-            A list containing nucleon coordinates as the values with the keys as the element's symbol.
-        spin: float
-            The total spin of the molecule system.
-        charge: int
-            The total charge of the molecule system.
-        seed: int, optional (default None)
-            Random seed to use for electron initialization.
-        batch_no: int, optional (default 10)
-            Number of batches of the electron's positions to be initialized.
-
-        """
-        # super(Ferminet, self).__init__()
-
+    Parameters:
+    -----------
+    nucleon_coordinates: List[List]
+      A list containing nucleon coordinates as the values with the keys as the element's symbol.
+    spin: int
+      The total spin of the molecule system.
+    ion_charge: int
+      The total charge of the molecule system.
+    seed_no: int, optional (default None)
+      Random seed to use for electron initialization.
+    batch_no: int, optional (default 10)
+      Number of batches of the electron's positions to be initialized.
+
+    Attributes:
+    -----------
+    nucleon_pos: np.ndarray
+        numpy array value of nucleon_coordinates
+    electron_no: np.ndarray
+        Torch tensor containing electrons for each atom in the nucleus
+    molecule: ElectronSampler
+        ElectronSampler object which performs MCMC and samples electrons
+    """
         self.nucleon_coordinates = nucleon_coordinates
         self.seed = seed
         self.batch_no = batch_no
         self.spin = spin
-        self.ion_charge = charge
-
-    def prepare_input_stream(self,) -> Tuple[Any, Any, Any, Any]:
```

Comment:
Why are we deleting this helper function?

Reply:
This helper function computes pairwise distances of each electron from nuclei and can be done in forward itself